specs/engage-digital_openapi3.yaml

```diff
@@ -735,6 +735,24 @@ paths:
           required: false
           schema:
             type: integer
+        - description: Only on Engage Messaging sources that are not on livechat mode. Enable or disable the source on Android.
+          in: query
+          name: enable_android
+          required: false
+          schema:
+            type: boolean
+        - description: Only on Engage Messaging sources that are not on livechat mode. Enable or disable the source on iOS.
+          in: query
+          name: enable_ios
+          required: false
+          schema:
+            type: boolean
+        - description: Only on Engage Messaging sources that are not on livechat mode. Enable or disable the source on the web.
+          in: query
+          name: enable_web
+          required: false
+          schema:
+            type: boolean

```

Comment:
I don't provide the `live_chat` field in the PUT because it is techteam only.

Reply:
Makes sense :+1: